simple-backend/nlpviewer_backend/urls.py

```diff
@@ -44,10 +48,19 @@
     path('documents/<int:document_id>/links/<int:link_id>/delete',
          document.delete_link),
 
+
+    path('projects', project.listAll),
+    path('projects/new', project.create),
+    path('projects/<int:project_id>', project.query),
+    path('projects/<int:project_id>/docs', project.query_docs),
+    path('projects/<int:project_id>/delete', project.delete),
+
     path('documents/<int:document_id>/text/edit', document.edit_text),   
 
-    path('nlp/load/<model_name>', nlp.load_model),
-    path('nlp/<int:document_id>/<model_name>', nlp.run_pipeline),
+    path('ontology_from_doc/<int:document_id>', document.get_doc_ontology_pack),
+
+    # path('nlp/load/<model_name>', nlp.load_model),
+    # path('nlp/<int:document_id>/<model_name>', nlp.run_pipeline),
```

Comment:
Coding style: commented discarded code can be removed during PR.

Reply:
The problem is the backend cannot start without forte and pytorch, is this by design?

Reply:
The current master has solved this issue. Now there will only be warnings if Forte is not installed.

Reply:
So now should I delete them for the PR? or uncomment them? And the same problem for the above, 'import'

Reply:
You can uncomment them now.